s2wav/tiling.py

```diff
@@ -0,0 +1,63 @@
+"""Script defining basic mathematical functions needed to carry out the tiling of the harmonic space. 
+"""
+import numpy as np
+import pytest
+
+
+def tiling_integrand(t: float, lam: float) -> float:
+    """Tiling integrand for wavelets. Intermediate step used to compute the wavelet 
+        and scaling function generating functions.
+
+    Args:
+        t (float): Real argument over which we integrate.
+        lam (float): Wavelet parameter which determines the scale factor between consecutive wavelet scales.
+        
+    Returns:
+        Value of tiling integrand for given t and scaling factor.
```

Comment:
All these docstrings look fantastic, the only thing is for returns we should have

`data_type: description of data`

so something like 

`float: Value of tiling integrand for a given t and scaling factor`

We should make sure this is consistent throughout and going forward :)